package.json

```diff
@@ -1,13 +1,13 @@
 {
   "name": "@devoinc/prettier-config",
-  "version": "1.0.0",
+  "version": "1.1.0",
   "description": "Simple Prettier config for Devo.",
   "main": "index.json",
   "author": "Devo dev team",
   "license": "MIT",
   "files": [
     "index.js"
```

Comment:
Let's remove this, while at it, shouldn't we?

Reply:
(I mean the field `files` altogether)

Reply:
Done!